resources/js/app.js

```diff
@@ -1,3 +1,5 @@
+---
+---
```

Comment:
why are these being added?

Reply:
@jayvdb This is the frontmatter needed by jekyll to interpolate a file. Without this we cannot read the value `gsocSwitchMonth` from the `_config.yml` file as suggested by @li-boxuan.

Reply:
To avoid this @aedorado you can pass the value to function `getGsocYear()` from html page itself.
Doing so, will omit those two new lines containing hyphen

Reply:
Yes please.  embedding jekyll liquid variables inside JS is just too confusing ;-)